tools/bin/ecflow/processes/BC/meteorological/metsim_metgrid.ecf

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+%include <qsub.h>
+%include <head.h>
+%manual
+Updates MetSim configuration file and runs MetSim to disaggregate forcings 
+%end
+
+echo "Today is $(date)"
+
+source activate monitor_py3
```

Comment:
Since you are using different conda environments for the ecflow scripts, it might be useful to add a brief comment as to why this is.